src/app/shared/pipes/timestamp.pipe.ts

```diff
@@ -0,0 +1,29 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'timestamp'
+})
+export class TimestampPipe implements PipeTransform {
```

Comment:
The closest I have found is `DatePipe`. It might be sufficient for our needs, but doesn't match what the this implementation does (allowing the hour count to go beyond 23).